packages/app/obojobo-document-json-parser/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "obojobo-document-engine": "^4.0.1"
+  },
+  "devDependencies": {
+    "lint-staged": "^6.1.1"
+  },
+  "name": "json-to-xml-parser",
```

Comment:
name needs to match the directory name as we'll publish this to npm. needs to be `obojobo-document-json-parser`